platform-manageability-agent/Makefile

```
@@ -0,0 +1,99 @@
+# SPDX-FileCopyrightText: (C) 2025 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+NAME ?= platform-manageability-agent
+GO_MOD ?= readonly
+BUILD_DIR ?= build/artifacts
+PACKAGE_BUILD_DIR ?= $(BUILD_DIR)/package
+PREFIX ?= /usr/local
+VERSION := $(shell cat VERSION)
+COMMIT := $(shell git rev-parse --short HEAD)
+ifneq (,$(findstring dev,$(VERSION)))
+	PKG_VERSION = $(VERSION)-$(COMMIT)
+else
+	PKG_VERSION = $(VERSION)
+endif
+TARBALL_DIR := $(BUILD_DIR)/$(NAME)-$(PKG_VERSION)
+REGISTRY := 080137407410.dkr.ecr.us-west-2.amazonaws.com
+
+SHELL := /bin/bash
+.SHELLFLAGS := -e -c
+
+ifeq ("$(wildcard ./common.mk)","")
+	ifeq ("$(wildcard ../common.mk)","")
+		$(error Neither ./common.mk nor ../common.mk found. Please provide a common.mk file.)
+	else
+		include ../common.mk
+	endif
+else
+	include ./common.mk
+endif
+
+# Documented targets:
+#   all:        Build, lint, and test the agent
+#   pmabuild:   Build the agent binary
+#   pmabuild_with_race: Build with Go race detector
+#   pmainstall: Install the agent binary to $(PREFIX)/bin
+#   package:    Build a Debian package (requires dh_make, dpkg-buildpackage)
+#   tarball:    Create a tarball using common-tarball
+#   clean:      Remove build artifacts
+#   test:       Run unit tests (from common.mk)
+#   fuzztest:   Run fuzz tests (from common.mk)
+
+.PHONY: all build clean help lint package test fuzztest tarball deb-push integration_test
+
+all: build lint test
```

```suggestion
all: build test
```
Since this is used by the `make package` command as part of the Debian build process, `lint` does not need to be included